billy-core/src/test/java/com/premiumminds/billy/core/test/services/builders/TestBusinessBuilder.java

```diff
@@ -44,7 +43,7 @@ public void doTest() {
 
         Mockito.when(this.getInstance(DAOBusiness.class).getEntityInstance()).thenReturn(new MockBusinessEntity());
 
-        Mockito.when(this.getInstance(DAOContext.class).get(Mockito.any(UID.class)))
+        Mockito.when(this.getInstance(DAOContext.class).get(Mockito.any()))
```

Comment:
Just to point out this erased validation in all tests by not specifying StringID.class